src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -74,6 +74,15 @@ public function __construct(
             );
         }
 
+        if ($this->fileLocator === null) {
+            trigger_deprecation(
+                'sylius/core-bundle',
+                '1.13',
+                'Not passing a $fileLocator to %s constructor is deprecated and will be removed in Sylius 2.0.',
```

Comment:
Some other deprecation notices specify the position of a parameter in the constructor, though I see the current message is in line with already existing notices in the file.